torch/_inductor/codegen/common.py

```diff
@@ -339,8 +338,8 @@ def python_argdefs(self):
             precompile_args.append(TensorArg(inner, outer, V.graph.get_dtype(outer)))
         for outer, inner in self.sizevars.items():
             arg_defs.append(inner)
-            call_args.append(outer)
-            precompile_args.append(SizeArg(inner, sympy_symbol(outer)))
+            call_args.append(str(outer))
+            precompile_args.append(SizeArg(inner, outer))
```

Comment:
I don't really understand this change lol

Reply:
Basically, inner/outer were previously normalized to strings, and then we relied on `sympy_symbol` to turn them back into the corresponding sympy symbol. This was (later) then passed into `size_hint` to get the shape somewhere.

Basically, previously:
```
outer: str => (call_args: str, precompile_args: sympy.Symbol)
```
Now
```
outer: sympy.Symbol => (call_args: str, precompile_args: sympy.Symbol)
```